server_admin/topics/sessions/transient.adoc

```diff
@@ -0,0 +1,11 @@
+[[_transient-session]]
+
+=== Transient sessions
+
+{project_name} has concept of transient sessions. When transient sessions are used, there is no real user session created after successful authentication.
+There is just temporary "transient" session created for the scope of the current request, which successfully authenticated user. This allows
+to run <<_protocol-mappers, protocol mappers>> after the authentication.
+
```

Comment:
I suggest
Only a temporary, or transient, session is created for the scope of the current request that successfully authenticated the user. This transient session allows {project_name} 
to run <<_protocol-mappers, protocol mappers>> after the authentication.